File: berufsschule/11/AP/CPP/Hausaufgaben/3_OnlineBuch_Java/src/Buch.java
import java.util.*;
public class Buch 
{
	private String titel;
	private Date erscheinungsjahr;
	
	public String getTitel() 
	{
		return titel;
	}
	public void setTitel(String titel) 
	{
		this.titel = titel;
	} 
	public String getErscheinungsjahr() 
	{
		return erscheinungsjahr.getDay() + "." + erscheinungsjahr.getMonth() + "." + erscheinungsjahr.getYear();
	}
	public void setErscheinungsjahr(Date erscheinungsjahr) 
	{
		this.erscheinungsjahr = erscheinungsjahr;
	}
	public void Navigation()
	{
		Inhaltsverzeichnis.Navigation();
	}
}

File: berufsschule/11/AP/CPP/Hausaufgaben/3_OnlineBuch/Buch.cpp
#include "Buch.h"

Buch::Buch()
{
	erscheinungsjahr = 1011970;
	titel = "titel";
}
Buch::~Buch()
{
}
char* Buch::getTitel()
{
	return titel;
}
void Buch::setTitel(char *name)
{
	titel = name;
}
int Buch::getJahr()
{
	return erscheinungsjahr;
}
void Buch::setJahr(int jahr)
{
	erscheinungsjahr = jahr;
}

